src/validator.js

```diff
@@ -87,12 +94,23 @@ export default class Validator {
     const error = Validator.rules[ruleName](field, value, options);
 
     if (error) {
-      return { field, rule: error, value, config: this.config[field] };
+      return this.createError(field, error, value, this.config[field]);
     }
 
     return error;
   }
 
+  createError(field, error, value, config) {
```

Comment:
What do you think about moving this to a function outside the class to keep it "private"?